opentracing/harness/api_check.py

```diff
@@ -121,9 +125,8 @@ def test_trace_attributes(self):
 
     def test_text_propagation(self):
         with self.tracer().start_trace(operation_name='Bender') as span:
-            dict_tracer_state, dict_attrs = self.tracer().propagate_span_as_text(
-                span=span
-            )
+            dict_tracer_state, dict_attrs = self.tracer(). \
+                propagate_span_as_text(span=span)
```

Comment:
I was just de-linting mechanically... I will change this and a bunch of other stuff once we've converged on the rest of this. I also need to update README.md's examples...